packages/content/src/Application/ContentNormalizer/Normalizer/ExcerptNormalizer.php

```diff
@@ -13,20 +13,31 @@
 
 namespace Sulu\Content\Application\ContentNormalizer\Normalizer;
 
+use Sulu\Content\Domain\Model\DimensionContentInterface;
 use Sulu\Content\Domain\Model\ExcerptInterface;
+use Sulu\Page\Domain\Model\PageInterface;
 
 class ExcerptNormalizer implements NormalizerInterface
 {
     public function enhance(object $object, array $normalizedData): array
     {
-        if (!$object instanceof ExcerptInterface) {
+        if (!$object instanceof ExcerptInterface || !$object instanceof DimensionContentInterface) {
             return $normalizedData;
         }
 
         $normalizedData['excerptTags'] = $normalizedData['excerptTagNames'];
         unset($normalizedData['excerptTagNames']);
         $normalizedData['excerptCategories'] = $normalizedData['excerptCategoryIds'];
         unset($normalizedData['excerptCategoryIds']);
+        $normalizedData['excerptAudienceTargetGroups'] = $normalizedData['excerptAudienceTargetGroupIds'] ?? [];
+        unset($normalizedData['excerptAudienceTargetGroupIds']);
+
+        $resource = $object->getResource();
+        if ($resource instanceof PageInterface) {
```

Comment:
ContentBundle must be decoupled from the other bundles

Reply:
move the normalizer to the PageBundle